src/api/auth.js

```diff
@@ -79,15 +79,15 @@ class AuthInterface {
   constructor(app) {
     this.app = app;
     this._registered = false;
-    this.setupUri();
+    this.setupURI();
   }
 
   /**
   * @private
   * Generate the app's URI for the IPC protocol using the app's id
   * and register the URI scheme.
   */
-  setupUri() {
+  setupURI() {
```

Comment:
I wouldn't do this change mainly because in this particular case it would break the camelCase convention/style we follow for JS and we are not really aligning it with any of the native lib functions I think. Thus I would instead change the `fromAuthURI` function we have in `index.js`.